apps/native/app/src/ui/lib/label/label.tsx

```diff
@@ -1,20 +1,26 @@
-import { dynamicColor, font } from '@ui/utils'
+import { dynamicColor } from '@ui/utils'
 import { Image } from 'react-native'
 import styled, { DefaultTheme } from 'styled-components/native'
 import dangerIcon from '../../assets/alert/danger.png'
 import infoIcon from '../../assets/alert/info-alert.png'
 import warningIcon from '../../assets/alert/warning.png'
+import { Typography } from '../typography/typography'
 
 type LabelColor = 'default' | 'primary' | 'danger' | 'warning'
-type HelperProps = { theme: DefaultTheme; color: LabelColor }
+type HelperProps = {
+  theme: DefaultTheme
+  color: LabelColor
+  blackTextColor?: boolean
+}
 
 interface LabelProps {
   color?: LabelColor
   icon?: React.ReactNode | boolean
   children?: React.ReactNode
+  blackTextColor?: boolean
```

Comment:
~This property pollutes the component in my mind. Could this not be used for every label that uses danger?

Reply:
Can't we just add another label color called urgent?

Reply:
We cannot use it for all danger alerts since we have multiple danger alerts that we want to keep the red color, but I could add a variant called urgent 💡